docs/guide/static-deploy.md

```diff
@@ -322,7 +322,7 @@ You can deploy your Vite app as a Static Site on [Render](https://render.com/).
 
 4. Specify a project name and branch.
 
-   - **Build Command**: `npm run build`
+   - **Build Command**: `npm install && npm run build`
```

Comment:
Note for others: `npm install` is required
https://docs.render.com/deploy-sveltekit#manual-deploy